app/graphql/users/resolvers.js

```diff
@@ -9,5 +9,26 @@ exports.userRegister = (parent, { userFields: user }) => {
     .then(userCreated => {
       logger.info(`User ${userCreated.firstName} ${userCreated.lastName} created successfully`);
       return userCreated;
+    })
+    .catch(error => {
+      logger.error(`Error trying to create user. Details: ${JSON.stringify(error)}`);
```

Comment:
I think you should forward the error to the client here, otherwise they won't know what went wrong.